index.html

```diff
@@ -1,90 +0,0 @@
-<!DOCTYPE html>
```

Comment:
I realized that shortly after I contacted Vyshakh about where the live files were actually located.